crates/bitwarden/src/platform/client_fido.rs

```diff
@@ -0,0 +1,47 @@
+use std::sync::{Arc, Mutex};
+
+use bitwarden_fido::{
+    Fido2Authenticator, Fido2Client, Fido2CredentialStore, Fido2UserInterface,
+    FidoEncryptionSettingStore,
+};
+
+use crate::Client;
+
+pub struct ClientFido2<'a> {
+    #[allow(dead_code)]
+    pub(crate) client: &'a Client,
+}
+
+impl FidoEncryptionSettingStore for Client {
+    fn get_encryption_settings(
+        &self,
+    ) -> Result<Arc<dyn bitwarden_crypto::KeyContainer>, bitwarden_core::VaultLocked> {
+        Ok(self.get_encryption_settings()?)
+    }
+}
```

Comment:
Temporary trait to solve a circular dependency, when we move `Client` to core this shouldn't be needed.